src/app/features/features.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-features',
+  templateUrl: './features.component.html',
+  styleUrls: ['./features.component.scss']
+})
+export class FeaturesComponent implements OnInit {
+  constructor() { }
+
+  ngOnInit() {
+    console.log("hello")
```

Comment:
Not needed

Reply:
Yes, I forgot to remove that, it was just when I was testing something else

Reply:
> Not needed

Yes, I forgot to remove that, it was just when I was testing something else